### PR TITLE
README: add note how people can complain about abusive behavior

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
README: add note how people can complain about abusive behavior

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

**Comment by Andersson007** [comment]
As it's described in https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations , I'll close it, thanks

**Comment by Andersson007** [comment]
thanks @Jorge-Rodriguez for the better wording!

**Comment by Andersson007** [comment]
ready for review

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ https://dev.azure.com/ansible/community.postgres/_apis/build/status/CI?branchNam
 
 We follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within this project.
```
Comment (Jorge-Rodriguez) [inline README.md:8]:
```suggestion
We follow the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within this project.
```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ https://dev.azure.com/ansible/community.postgres/_apis/build/status/CI?branchNam
...
 
+If you encounter abusive behavior violating the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html), please refer to the [policy violations](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations) section of the Code of Conduct for information on how to raise a complaint.
+
 ## Contributing to this collection
 
 The content of this collection are made by [people](CONTRIBUTORS) like you.
```
Comment (Jorge-Rodriguez) [inline README.md:14]:
```suggestion
The contents of this collection are made by [people](CONTRIBUTORS) like you.
```

**Review by Jorge-Rodriguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md]: good spot, thanks! committed

File: README.md (hunk 2 above)
Comment (Andersson007) [reply README.md]: It doesn't allow to merge it here, so I'll create a new PR, thanks!

**Comment by Andersson007** [comment]
@Jorge-Rodriguez thanks for reviewing! My congratulations on becoming a community.postgresql contributor:)

**Pull request merged**